bin/fix-bin.cjs

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
+const { chmod } = require('node:fs/promises');
+const { join } = require('node:path');
+const process = require('node:process');
+
+(async () => {
+  if (process.platform !== "win32") {
+    await chmod(join(__dirname, '../dist/cli/jhipster.cjs'), 0o755);
```

Comment:
```suggestion
    await chmod(join(__dirname, '../dist/cli/jhipster.cjs'), 0755);
```

Reply:
The octal syntax should work correctly. Is a preference?

Reply:
Weel seems we cannot use octal
![image](https://github.com/jhipster/generator-jhipster/assets/3706415/a9572b2d-a443-4d71-bf89-c865f7975ece)